wire-schema/src/main/java/com/squareup/wire/schema/internal/parser/ProtoParser.java

```diff
@@ -341,23 +341,22 @@ private FieldElement readField(
     reader.require('=');
     int tag = reader.readInt();
 
-    FieldElement.Builder builder = FieldElement.builder(location)
-        .label(label)
-        .type(type)
-        .name(name)
-        .tag(tag);
-
     List<OptionElement> options = new OptionReader(reader).readOptions();
     reader.require(';');
 
     options = new ArrayList<>(options); // Mutable copy for extractDefault.
     String defaultValue = stripDefault(options);
 
     documentation = reader.tryAppendTrailingDocumentation(documentation);
-    return builder.documentation(documentation)
-        .defaultValue(defaultValue)
-        .options(ImmutableList.copyOf(options))
-        .build();
+    return new FieldElement(
+        location,
+        label,
+        type,
+        name,
+        defaultValue,
+        tag,
+        documentation,
+        ImmutableList.copyOf(options));
```

Comment:
Guava 🚫 Not sure if you're doing now or follow-up

Reply:
replaced with `new ArrayList()`.  Eventually, will be nuking the other Guava bits.